.github/workflows/_test_slurm_pyxis.yaml

```diff
@@ -2,29 +2,11 @@ name: ~test multi-node jobs via SLURM+Pyxis
 
 on:
   workflow_call:
-    secrets:
```

Comment:
Why would you remove it? It will make the workflow less adaptable to more clusters.